scripts/core/data/index.ts

```diff
@@ -32,6 +32,10 @@ addWebsocketEventListener(
         const RESOURCE_BLACKLIST = [
```

Comment:
IMO, this should rather be an `allow list` rather than a `deny list`.
I am seeing requests to the server that are not needed, with the `UPDATE_ENTITY` dispatched to redux.

![image](https://user-images.githubusercontent.com/22425232/122724410-3e3d8780-d2b7-11eb-8222-dd1044b80c17.png)

Reply:
yep makes sense

Reply:
@tomaskikutis can you make that change in this PR?

Reply:
Sure